python-threatexchange/threatexchange/signal_type/tests/test_pdq_index2.py

```diff
@@ -0,0 +1,195 @@
+import pytest
+import io
+import faiss
+from threatexchange.signal_type.pdq.pdq_index2 import (
+    PDQIndex2,
+    _PDQHashIndex,
+    DIMENSIONALITY,
+    DEFAULT_MATCH_DIST,
+)
+
+
+@pytest.fixture
+def empty_index():
+    """Fixture for an empty index."""
+    return PDQIndex2[str]()
```

Comment:
blocking: Why subscript this?
```suggestion
    return PDQIndex2()
```